src/propmcwf.jl

```diff
@@ -0,0 +1,95 @@
+immutable QuMCWF <: QuPropagatorMethod
+    solver::QuPropagatorMethod
```

Comment:
Could you do this via a `Dict` like for the other solvers (i.e. `options::Dict{Symbol, Any}`)?